tests/ffi/memory/test.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -eux
+
+# shellcheck disable=SC1091
+
+. ../memory/prepare.sh
+
+prepare_test
+reload_config
+
+# https://quay.io/repository/centos-sig-automotive/ffi-tools:latest
+podman run --replace -d --name ffi-asil localhost/ffi-tools:latest \
```

Comment:
Broken container
https://gitlab.com/CentOS/automotive/container-images/-/tree/main/images/ffi-tools

MR is waiting for fix
@odra

Reply:
Done